src/main/permissionsManager.ts

```diff
@@ -152,6 +153,11 @@ export class PermissionsManager extends JsonFileManager<PermissionsByOrigin> {
             return false;
         }
 
+        // For GPO servers, we always allow permissions since they are trusted
+        if (Config.registryData?.servers?.some((s) => parseURL(s.url)?.href === serverURL.href)) {
```

Comment:
Can `serverURL` be `undefined` here? Is it safe to access the `href` property?

Reply:
Nope, see the above falsy check: https://github.com/mattermost/desktop/pull/3231/files#diff-fc4629b25400b7a5efa825a58b4db1a3caaba2c20770948a5ba12f6b8de0b8eaR152

Reply:
Ahh - sorry for the noise. Was hidden behind a collapsed region :+1:

Reply:
Though for some reason `tsc` agreed with you :P